add test metrics in ci capability (#5774)
fishtown-analytics/docs.getdbt.com: website/docs/docs/use-dbt-semantic-layer/sl-faqs.md

Yes, while [entities](/docs/build/entities) must be defined under “entities,” they can be queried like dimensions in downstream tools. Additionally, if the entity isn't used to perform joins across your semantic models, you may optionally define it as a dimension.
</Expandable>

<Expandable alt_header="Can I test my semantic models and metrics?">

> Yes! You can test your metrics in a few ways:

Author: Added to faqs

> Yes! You can test your metrics in a few ways:

Author: ```suggestion
Yes! You can validate your semantic nodes (semantic models, metrics, saved queries) in a few ways:
```

- [Query and validate you metrics](/docs/build/metricflow-commands) in your development tool before submitting your code changes.

> - [Test and validate your metrics in CI](/docs/deploy/ci-jobs#test-metrics-in-ci) to ensure code changes made to dbt models don't break these metrics.

Author: ```suggestion
- [Validate semantic nodes in CI](/docs/deploy/ci-jobs#semantic-validations-in-ci) to ensure code changes made to dbt models don't break these metrics.
```
